rest-api/recipe/controllers/recipe-controllers.js

```diff
@@ -1,9 +1,31 @@
+const dbRecipe = require('../models/recipe-models')
+
+module.exports = {
+  getRecipes,
+  getRecipeById
+};
+
 async function getRecipes(req, res) {
   try {
-    res.status(200).json({ message: 'up an running' });
+    const recipes = await dbRecipe.getRecipes();
+    if (recipes.length) {
+      res.status(200).json(recipes);      
+    }
+    else {
+      res.status(404).json({message: 'There are no saved recipes'})
+    }
   } catch (error) {
-    res.status(500).json({ message: 'fell flat on my face' });
+    res.status(500).json({ message: 'There was an error retrieving the saved recipes' });
```

Comment:
Look at the user routes on how to output 500, it should include the initial error

Reply:
Yes it should, I missed that. Thanks for point that out